tox.ini

```diff
@@ -142,16 +143,27 @@ deps = {[with-debug-tools]deps}
 # --------------------------------------------------------------------
 # mypy
 # --------------------------------------------------------------------
+
 [testenv:mypy]
+skipsdist = True
+skip_install = True
+deps = mypy
+       types-jsonschema
+commands = python {toxinidir}/tools/clean.py "{toxinidir}/falcon"
+           mypy falcon
+
+[testenv:mypy_tests]
 deps = {[testenv]deps}
-       pytest-mypy
+       mypy
        types-requests
        types-PyYAML
        types-ujson
        types-waitress
        types-aiofiles
+       types-jsonschema
 commands = python "{toxinidir}/tools/clean.py" "{toxinidir}/falcon"
-           pytest --mypy -m mypy tests []
+           mypy e2e-tests/server
+           mypy tests
```

Comment:
I kinda liked that it would appear as a pytest output but I'm fine either way

Reply:
I had issues with that plugin, it was concealing errors in some cases, thought it would be much less maintenance burden to stay with vanilla `mypy` which is very widely used compared to running through the said plugin.

Reply:
Understood